src/cls/IPM/Storage/SystemRequirements.cls

```diff
@@ -19,11 +19,15 @@ Method VersionStringSet(tValue) As %Status
 
 Property Interoperability As %String(VALUELIST = ",enabled,disabled", XMLPROJECTION = "ATTRIBUTE");
 
+Property Health As %Boolean(XMLPROJECTION = "ATTRIBUTE");
```

Comment:
Nit: Worth having an InitialExpression of 0

Reply:
I disagree - this is tri-state, 1 means required, 0 means disallowed, null means we don't care.

Reply:
(perhaps could use a comment to that effect though.)